### ISSUE TITLE
pycodestyle cannot be disabled

### ISSUE THREAD

**Issue by jedix** [steelbrain/linter#1716]
I installed the following packages:

* linter
* linter-flake8
* linter-ui-default

After configuring linter-flake8 with a `maxLineLength` of 119 characters, I expected no more warnings:

```
  "linter-flake8":
    maxLineLength: 119
 ```

But warnings are still displayed, using the provider "pycodestyle":

![grafik](https://user-images.githubusercontent.com/211924/98913267-6d8d4a00-24c7-11eb-8996-07b78aa9d6b2.png)

I tried to disable this provider:

```
  linter:
    disabledProviders: [
      "pycodestyle"
    ]
```

Which had no effect.

Am I doing something wrong?

**Comment by aminya** [comment]
Do you have ide-python installed?

**Comment by jedix** [comment]
Yes, I have. Thanks for the hint. That is where the warnings are coming from.
